### DISCUSSION TITLE
How to Add tags to all the resources that are being created

### DISCUSSION THREAD

**Discussion by ismailcool5152** [CloudSnorkel/cdk-github-runners#644] [Q&A] [answered]
Hi Guys, 

we are trying to deploy this solution in prod but it seems by default there are no tags being added to the resources. 

Is there a way we can add Tags to all the resources which are being created ?

**Comment by kichik** [comment]
Have you tried using `cdk.Tags.of(runner).add()`?

**Reply by ismailcool5152** [reply]
Hi Kichik, 

I am not sure in which exact file or at what time , i need to use above command . 

I am using TS to define my resources and then i am using 

npm run build
npx cdk deploy --tags uai=uai3064620 --require-approval never

To deploy the stack. At what point and at what file we need to add cdk.Tags.of(runner).add()
 And how ? Any example that you could provide ?

**Reply by kichik** [reply, marked as answer]
I don't think `--tags` is a parameter that [`cdk deploy`](https://docs.aws.amazon.com/cdk/v2/guide/ref-cli-cmd-deploy.html) accepts. Here is a basic example with TypeScript:

```
import * as cdk from 'aws-cdk-lib';
import { App, Stack } from 'aws-cdk-lib';
import { GitHubRunners } from '@cloudsnorkel/cdk-github-runners';

const app = new App();
const stack = new Stack(app, 'github-runners');
const runners = new GitHubRunners(stack, 'runners');
cdk.Tags.of(runner).add('mytag', 'myvalue');

app.synth();
```

More info about tags in CDK here https://docs.aws.amazon.com/cdk/v2/guide/tagging.html

**Reply by ismailcool5152** [reply]
Yes, this worked. 

Thanks for the help :)
